playlists/excel.yaml

```diff
@@ -20,6 +20,17 @@
   group: Basics
```

Comment:
No, go ahead and commit it.  I think it's useful to see for diffing's sake, and there is no harm (Travis will just override it anyway)